charts/enterprise/traefik/questions.yaml

```diff
@@ -16,6 +16,7 @@ questions:
 
 # Include{containerConfig}
 # Include{podOptions}
+# Include{crdsTrue}
```

Comment:
Do we have something on common side that checks `.Values.crds`?

Reply:
this was a plan, but has been scratched.

Reply:
```suggestion
```